ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.view.js

```diff
@@ -179,8 +179,10 @@ module.exports = Marionette.LayoutView.extend({
       error: () => {
         this.errorMessage()
       },
+      wait: true,
```

Comment:
❓ why did you need the wait?

Reply:
This is waiting for the sent out request to backend to respond before adding the Result Form. Without the wait, it immediately adds the Result Form box without the content inside, and if it returns with an error, you are just left with that empty box.